docs/source/features/passes/pytorch.md

```diff
@@ -2,6 +2,33 @@
 
 PyTorch is an optimized tensor library for deep learning using GPUs and CPUs.
 
+## QLoRA
+`QLoRA` is an efficient finetuning approach that reduces memory usage by backpropagating gradients through a frozen, 4-bit quantized pretrained model into Low Rank Adapters (LoRA). It is based on
+the QLoRA [paper](https://arxiv.org/abs/2305.14314) and [code](https://github.com/artidoro/qlora/blob/main/qlora.py). More information on LoRA can be found in the [paper](https://arxiv.org/abs/2106.09685).
+
+The output model is the input transformers model along with the quantization config and the fine-tuned LoRA adapters. The adapters can be loaded and/or merged into the original model using the
+`peft` library from Hugging Face.
+
+This pass only supports Hugging Face transformers PyTorch models.
+
+**Note:** QLoRA requires a GPU to run.
+
+### Example Configuration
+```json
+{
+    "type": "QLoRA",
+    "config": {
+        "compute_dtype": "bfloat16",
+        "quant_type": "nf4",
+        "train_data_config": ...,
+        "training_args": {
+            "learning_rate": 0.0002,
+            ...
```

Comment:
>... [](http://example.com/codeflow?start=12&length=3)

will this emit doc build warning like [https://github.com/microsoft/Olive/pull/485](https://github.com/microsoft/Olive/pull/485)?

Reply:
Done! Added hf training args to passes api reference and linked to it.